core/domain/exp_services.py

```diff
@@ -361,38 +469,72 @@ def get_exploration_ids_matching_query(query_string, cursor=None):
 def get_non_private_exploration_summaries():
     """Returns a dict with all non-private exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        dict. The keys are exploration ids and the values are corresponding 
+        non-private ExplorationSummary domain objects.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_non_private())
 
 
 def get_top_rated_exploration_summaries(limit):
-    """Returns a dict with top rated exploration summary domain objects,
+    """Returns a dict with top rated exploration summary model instances,
     keyed by their id.
+
+    Args:
+        limit: int. Number of exploration summary model instances to be
+            returned.
+
+    Returns:
+        dict. The keys are exploration ids and the values are corresponding
+        top rated ExplorationSummary domain model instances.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_top_rated(limit))
 
 
 def get_recently_published_exp_summaries(limit):
-    """Returns a dict with all featured exploration summary domain objects,
+    """Returns a dict with all featured exploration summary model instances,
     keyed by their id.
+
+    Args:
+        limit: int. Number of exploration summary model instances to be
+            returned.
+
+    Returns:
+        dict. The values are all featured ExplorationSummary model instances, 
+        keyed by their exploration id.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_recently_published(limit))
 
 
 def get_all_exploration_summaries():
-    """Returns a dict with all exploration summary domain objects,
+    """Returns a dict with all exploration summary model instanes,
     keyed by their id.
+
+    Returns:
+        dict. A dict with all ExplorationSummary model instances keyed by their
+        exploration id.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_all())
 
 
 # Methods for exporting states and explorations to other formats.
 def export_to_zip_file(exploration_id, version=None):
-    """Returns a ZIP archive of the exploration."""
+    """Returns a ZIP archive of the exploration.
+
+    Args:
+        exploration_id: str. The id of the exploration to export.
+        version: int or None. If provided, this indicates which version of
+            the exploration to export. Otherwise, the latest version of the 
+            exploration is exported.
+
+    Returns:
+        A ZIP archive of the exploration.
```

Comment:
What is the type of the return value? (Please check using, e.g., logging.info()).